content/api/api-notifications.md

```diff
@@ -7,68 +7,600 @@ author: Lawrence Lane
+
+
+
+### Request URL
+
+`https://app.metricly.com/notifications/{id}`
+
+### CURL
+
+In the following CURL example, only the **notificationId** must be included with the Request URL.
+
+```
+curl -X DELETE --header 'Accept: */*' --header 'User-Agent: none' 'https://app.metricly.com/notifications/84005'
+
+```
+
+### Response Body
+
+The following response returns no content and a 204 success code.
+
+```
+no content
+```
+
+{{% /expand %}}
+
+---
+
+## GET from /notifications/{id}
+
+{{< button href="https://app.metricly.com/swagger-ui.html#!/notifications/getSingleUsingGET_2" theme="info" >}} GET {{< /button >}} Use this endpoint to get details on a specific notification.
+{{% expand "View Method Details." %}}
+
+### Parameters
+
+| Parameter | Parameter Type | Data Type | Description |
+|--------------------|----------------|---------------|------------------------------------------------------------------------------------|
+| id | path | long | Notification's unique ID.|
+
+
+### Request URL
+
+`https://app.metricly.com/notifications/{id}`
+
+### CURL
+
+In the following CURL example, only the **notificationId** must be included with the Request URL.
+
+```
+curl -X GET --header 'Accept: application/json' 'https://app.metricly.com/notifications/42119'
+
+```
+
+### Response Body
+
+The following response returns the full notification template.
+
+```
+{
+  "notification": {
+    "id": 42119,
+    "tenantId": "0fa5384a-3b36-4766-ad68-43e7c6af54f6",
+    "enabled": true,
+    "type": "email",
+    "properties": {
+      "templateType": "default",
+      "address": "test.email@gmail.com",
+      "payloadType": "default",
+      "bodyTemplate": "<#if payloadType == \"event\">\n  ${eventCategory.name}\n</#if>\n<#if payloadType == \"event_cleared\">\n  CLEAR\n</#if>",
+      "name": "the test notification",
+      "subjectTemplate": "Metricly Event [${policyName}]",
+      "awsAuthentication": "role"
+    }
+  }
+}
+```
+
+{{% /expand %}}
+
+---
+
+
+
+## PUT to /notifications/{id}
+
+{{< button href="https://app.metricly.com/swagger-ui.html#!/notifications/replaceUsingPUT" theme="warning" >}} PUT {{< /button >}} Use this endpoint to update notifications.
+{{% expand "View Method Details." %}}
+
+### Parameters
+
+| Parameter | Parameter Type | Data Type | Description |
+|--------------------|----------------|---------------|------------------------------------------------------------------------------------|
+| User-Agent  | header  | string  | User-Agent  |
+| id | path | long | Notification's unique ID.|
+| Notification   | body  | JSON  |  JSON instructions to edit or build notifications. |
+
+
+### Request URL
+
+`https://app.metricly.com/notifications/{id}`
+
+### CURL
+
+In the following CURL example, the generic email notification used in previous examples is swapped for a Slack Webhook notification. Note that different notification types may have different minimum requirements. In this case, a Webhook must have a **url** property.
```

Comment:
```suggestion
In the following CURL example, the Slack notification **url** property is updated with a new URL because the old URL belongs to a Slack channel and integration that no longer exist.
```